sdk/kiosk/README.md

```diff
@@ -19,6 +18,27 @@ You can also use your preferred package manager, such as yarn or pnpm.
 
 Here are some indicative examples on how to use the kiosk SDK.
 
+<details>
+<summary>Find an addresses' owned kiosk(s)</summary>
```

Comment:
"Find an address" or "find addresses"
But maybe more clear to say "Find the kiosks owned by an address" or "Find the kiosks an address owns"